mmv1/templates/terraform/constants/network_services_gateway.go.tmpl

```diff
@@ -122,5 +131,9 @@ func deleteSWGAutoGenRouter(d *schema.ResourceData, config *transport_tpg.Config
 		return err
 	}
 
+	err = tpgcompute.ComputeOperationWaitTime(
```

Comment:
Should this be `NetworkServicesOperationWaitTime`? Why are we trying to pull in the compute function?

Reply:
It's using `ComputeOperationWaitTime` because the `Router` resource is under `Compute`:

```
routerPath := fmt.Sprintf("{{"{{"}}ComputeBasePath{{"}}"}}projects/{{"{{"}}project{{"}}"}}/regions/{{"{{"}}location{{"}}"}}/routers/%s", routerId)
```

Reply:
Ah, missed that, thanks!